src/parser/jobs/dnc/modules/Combos.ts

```diff
@@ -0,0 +1,25 @@
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {dependency} from 'parser/core/Module'
+import Combatants from 'parser/core/modules/Combatants'
+import CoreCombos, {AoeEvent} from 'parser/core/modules/Combos'
+import DirtyDancing from './DirtyDancing'
+
+const GCD_TIMEOUT_MILLIS = 15000
+
+export default class Combos extends CoreCombos {
+	// Override statics
+	static suggestionIcon = ACTIONS.CASCADE.icon
+
+	@dependency private dancing!: DirtyDancing
+
+	// Override check for allowable breaks. If two dances were started during the initial context's GCD timeout window,
+	// (ie, both Standard and Technical were danced), then we'll allow it
+	isAllowableComboBreak(event: AoeEvent, context: AoeEvent[]): boolean {
+		// Shouldn't ever be the case, but protect against weird shit
+		if (context.length !== 1) {
```

Comment:
geeeenerally avoid explicit numbers on arrays and similar if it's avoidable - `if (content.length < 1)`, and `dancesInRange(...) >= 2`. It won't nessicarily get hit, but it means it'll be handled sanely if it does.

Reply:
I was intentional about the explicit length check. Since DNC's combo is a two-step, the only valid context we should be getting is length === 1 so anything else is probably bad data (might even be worth flagging as a broken log...?). Its primary purpose though is to make sure the context[0].timestamp references later don't bomb if something odd _does_ sneak through.